internal/configs/virtualserver_test.go

```diff
@@ -866,13 +867,12 @@ func TestGenerateUpstreamWithKeepalive(t *testing.T) {
 func TestGenerateUpstreamForZeroEndpoints(t *testing.T) {
```

Comment:
looks like we need this test func anymore, because the logic for handling 0 endpoints for Plus/OSS was moved to a different function